### DISCUSSION TITLE
Two Hue instances for one entertainment zone

### DISCUSSION THREAD

**Discussion by gnisch** [awawa-dev/HyperHDR#38] [Q&A] [answered]
Hello 
Is it possible to have two instances connected to the same entertainment zone, one instance for bulbs, and one for hue playbars?

Tak btw to ja ziomek z peppera, to chyba ze mną rozmawiałeś. Jeszcze raz świetna robota, jesteś wielki :) Jakbyś potrzebował kogoś do testu to pisz.

**Comment by awawa-dev** [comment]
Cześć :) 
Never tried... did you try to create two entertainment zones in Philips app? Don't now if it's possible on same hub though. Then maybe we could assign one zone per instance.

**Comment by awawa-dev** [comment, marked as answer]
Currently Philips hue wizard in HyperHDR doesn't allow to select single device from entertainment zone: it uses all of them. Different zones configuration is only option for now (but it's untested).

Odezwę się  przez Peppera ;) jak nowa wersja będzie wymagała testów.
Pozdrawiam

**Reply by gnisch** [reply]
I'll try two zones. 

Będę czekać.
